maintenance_project/test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-base_maintenance @ git+https://github.com/OCA/maintenance.git@refs/pull/400/head#subdirectory=base_maintenance
```

Comment:
@Anxo82 test requirements file seems to be good, but it should be placed in the root folder, not inside addon directory.